src/genophenocorr/preprocessing/_vep.py

```diff
@@ -77,10 +81,12 @@ def annotate(self, variant_coordinates: VariantCoordinates) -> typing.Sequence[T
         """
         response = self._query_vep(variant_coordinates)
         annotations = []
+        if response is None:
+            self._logger.error('VEP did not finish successfully.')
```

Comment:
`_query_vep` always logs an error before returning `None`. This is, therefore, redundant and should be removed. If you feel the user need more context, then you should update the logging in the `_query_vep` method instead.